.gitlab-ci.yml

```diff
@@ -71,11 +70,13 @@ format:
   before_script:
     - docker login -u $CI_USER_NAME -p $GITLAB_ACCESS_TOKEN $CI_REGISTRY
   script:
-    #- docker run -v $(pwd):/project $CI_REGISTRY/r3/docker/julia-custom --project=@. -e 'using JuliaFormatter; format(".", verbose=true);'
-    - if [[ $(docker run -i -v $(pwd):/git alpine/git status -s | wc -l) -gt 0 ]]; then
-        GITHUB_COMMENT="There are files that need formatting :x:";
+    - |
+      docker run -v "$PWD":/project $CI_REGISTRY/r3/docker/julia-custom julia -e 'using JuliaFormatter; format("/project", verbose=true);'
+    - |
+      if [ `docker run -i -v "$PWD":/git alpine/git status -s | wc -l` -ne 0 ] ; then
+        GITHUB_COMMENT=":red_square: Commit ${CI_COMMIT_SHORT_SHA} requires formatting! :red_square:"$'\n\n'"Required formatting changes summary:"$'\n```'"`docker run -i -v \"$PWD\":/git alpine/git diff --stat`"$'\n```'
```

Comment:
somewhat this space `:red_square: Commit`  isn't taken into account properly

Reply:
can you also put this into several lines?

Reply:
the space between the emoji and "Commit" ? that one is there (at least I see it). Maybe it's disappearing because of some weird emoji behavior.